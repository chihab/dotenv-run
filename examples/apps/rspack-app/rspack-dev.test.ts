import { execSync } from "child_process";
import { describe, it, expect } from "vitest";

describe("dotenv-run should give the expected output", () => {
  it("dev", () => {
    const actual = execSync("node dist/dev/main.js", {
      encoding: "utf8",
    });
    expect(actual).toMatchSnapshot();
  });
});
