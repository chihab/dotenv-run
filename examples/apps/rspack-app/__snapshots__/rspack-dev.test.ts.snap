// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`dotenv-run should give the expected output > dev 1`] = `
"https://dotenv-run.dev/api/v1/users
development
"
`;
